blog/electron-28-0.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Electron 28.0.0
+date: 2023-11-28T00:00:00.000Z
+authors:
+  - name: ckerr
+    url: 'https://github.com/ckerr'
+    image_url: 'https://github.com/ckerr.png?size=96'
+slug: electron-28-0
+---
+
+Electron 28.0.0 has been released! It includes upgrades to Chromium `120.0.6099.56`, V8 `12.0`, and Node.js `18.18.2`.
+
+---
+
+The Electron team is excited to announce the release of Electron 28.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Twitter](https://twitter.com/electronjs) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+## Highlights
+
+- Implemented support for ECMAScript modules or ESM (What are ECMAScript modules? [learn more here](https://nodejs.org/api/esm.html#modules-ecmascript-modules). This includes support for ESM in Electron proper, as well as areas such as the `UtilityProcess` API entrypoints. See [#37535](https://github.com/electron/electron/pull/37535). and [#40047](https://github.com/electron/electron/pull/40047) for more details
```

Comment:
Instead of linking the PRs should we just link the docs? https://www.electronjs.org/docs/latest/tutorial/esm